system-addon/test/functional/mochitest/head.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
+
+function popPrefs() {
+  return SpecialPowers.popPrefEnv();
+}
+function pushPrefs(...prefs) {
+  return SpecialPowers.pushPrefEnv({set: prefs});
+}
+
+// Activity Stream tests expect it to be enabled, and make sure to clear out any
+// preloaded browsers that might have about:newtab that we don't want to test
+const ACTIVITY_STREAM_PREF = "browser.newtabpage.activity-stream.enabled";
+pushPrefs([ACTIVITY_STREAM_PREF, true]);
+gBrowser.removePreloadedBrowser();
+
+/**
+ * Helper to wait for potentially preloaded browsers to load
+ */
```

Comment:
Probably worth pointing out in the comment that this is effectively the "load" event, and contains no guarantee about what has or hasn't been rendered.